src/test/scala/org/broadinstitute/hail/io/compress/BGzipCodecSuite.scala

```diff
@@ -44,7 +44,7 @@ class TestFileInputFormat extends hd.mapreduce.lib.input.TextInputFormat {
 
 class BGzipCodecSuite extends SparkSuite {
   @Test def test() {
-    sc.hadoopConfiguration.set("io.compression.codecs", "org.apache.hadoop.io.compress.DefaultCodec,org.broadinstitute.hail.io.compress.BGzipCodec,org.apache.hadoop.io.compress.GzipCodec")
+    sc.hadoopConfiguration.setLong("mapreduce.input.fileinputformat.split.minsize", 1L)
```

Comment:
We now call `driver.configure` in the @BeforeClass method on SparkSuite.  This sets the codecs correctly, but also sets the min size to 1M.  The bgzip files generated in the test are smaller than 1M, so we need to remove the minimum to verify correct behavior